src/rgw/rgw_lc_s3.cc

```diff
@@ -67,6 +67,17 @@ bool RGWLifecycleConfiguration_S3::xml_end(const char *el) {
   return true;
 }
 
+bool LCFilter_S3::xml_end(const char* el) {
```

Comment:
Note that the whole xml parsing can be simplified by using `RGWXMLDecoder` (see `rgw_xml_enc.cc`)